sdk/src/lib.rs

```diff
@@ -109,6 +117,100 @@ impl SP1Prover {
         }
     }
 
+    pub async fn relay_remote(
+        access_token: String,
+        proof_id: &str,
+        chain_ids: Vec<u32>,
+        callbacks: Vec<&str>,
+        callback_datas: Vec<&str>,
```

Comment:
these should not be string, and they should be slice (not owned) instead of vec (owned), maybe it should be `&[[u8;32]]` and `&[Vec<u8>]`

Reply:
[u8; 20] but yeah sgtm, updated